packages/connect-form/src/components/connect-form-actions.tsx

```diff
@@ -11,37 +11,76 @@ import {
   ConnectionFormError,
   ConnectionFormWarning,
 } from '../utils/validation';
+import { ConnectionInfo } from 'mongodb-data-service';
 
 const formActionStyles = css({
   borderTop: `1px solid ${uiColors.gray.light2}`,
   paddingLeft: spacing[4],
   paddingRight: spacing[4],
 });
 
+const formActionButtonStyles = css({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  '& div:first-child': {
+    marginRight: spacing[2],
+  },
+});
+
 const formButtonsStyles = css({
   paddingTop: spacing[3],
   paddingBottom: spacing[3],
   textAlign: 'right',
 });
 
 function ConnectFormActions({
+  initialConnectionInfo,
   errors,
   warnings,
   onConnectClicked,
+  onSaveClicked,
+  saveDisabled,
 }: {
-  onConnectClicked: () => void;
+  initialConnectionInfo: ConnectionInfo;
```

Comment:
the `initialSomething` is used more for components that keep track of the state of "something" and then change it, for example, if there would be an "initial connection" and one that is edited from a `useState` hook. This component is stateless, I'd just call this parameter `connectionInfo`.